src/views/components/trace/trace-search.vue

```diff
@@ -132,16 +141,12 @@ limitations under the License. -->
     private maxTraceDuration: string = localStorage.getItem('maxTraceDuration') || '';
     private minTraceDuration: string = localStorage.getItem('minTraceDuration') || '';
     private instance: Option = { label: 'All', key: '' };
-    private endpointName: string = localStorage.getItem('endpointName') || '';
+    private endpoint: Option = { label: 'All', key: '' };
     private traceId: string = localStorage.getItem('traceId') || '';
     private traceState: Option = { label: 'All', key: 'ALL' };
     private tags: string = '';
     private tagsList: string[] = [];
-
     private created() {
-      this.endpointName = this.$route.query.endpointname
-        ? this.$route.query.endpointname.toString()
-        : this.endpointName;
```

Comment:
After delete this, we aren't able to visit a link with endpointName, link trace id. Should we keep this? @wu-sheng

Reply:
I think we only should keep `traceid` as the URI parameter for now.

Reply:
@Fine0830 Please confirm, after this change, whether endpoint id is not required in the query? If so, I am willing to remove it from query protocol if the CLI side is fine with that.

Reply:
Yes, the endpoint id is not required here.